### ISSUE TITLE
`CKV_GCP_69` needs to be updated to be compliant with the new provider changes around `workload_metadata_config` configuration block

### ISSUE THREAD

**Issue by calexandre** [bridgecrewio/checkov#1777]
**Describe the bug**
The code block `workload_metadata_config` has changed and checkov needs to follow up with an upgrade, so that `CKV_GCP_69` is compliant to the new model, as described here: https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/container_cluster#workload_metadata_config

**To Reproduce**

First update the google provider to `3.88.0` (in my case I bumped from `3.82.0 -> 3.88.0`).

The following will throw a deprecation notice on terraform's side:

```hcl
workload_metadata_config {
      node_metadata = "GKE_METADATA_SERVER" # https://docs.bridgecrew.io/docs/ensure-the-gke-metadata-server-is-enabled
    }
```

After updating the code to the new mode, checkov will throw a violation error and needs to be explicitly skipped;

```hcl
workload_metadata_config {
      #checkov:skip=CKV_GCP_69:Ensure the GKE Metadata Server is Enabled - false positive (waiting for checkov update) before was GKE_METADATA_SERVER and now is GKE_METADATA
      mode = "GKE_METADATA"
    }
```

**Expected behavior**
Checkov needs to be updated to be compliant with the new provider.

**Desktop (please complete the following information):**
 - OS: ubuntu (wsl)
 - Checkov Version `2.0.465`

**Comment by gruebel** [comment]
thanks for reporting, yeah this should definitely be updated.

**Comment by mzwennes** [comment]
@calexandre @gruebel Hopefully this will help: https://github.com/bridgecrewio/checkov/pull/1877

**Comment by gruebel** [comment]
thanks to @mzwennes it should be fixed in version `2.0.559` 🚀
